paddle/operators/nccl/nccl_gpu_common.cc

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserve.
```

Comment:
what is this file used for

Reply:
build an operator independent module, which may be used for `Multiexecutor` or some more module.